hack/push-and-tag.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+# File containing the list of images
+OUTPUT_TASK_BUNDLE_LIST="${OUTPUT_TASK_BUNDLE_LIST-${SCRIPTDIR}/../task-bundle-list}"
+
+# Read the file and process each line
+while IFS=, read -r original_image new_image; do
+    # Remove the quotes from the strings
+    original_image=$(echo "$original_image" | tr -d '"' | xargs)
+    new_image=$(echo "$new_image" | tr -d '"' | xargs)
```

Comment:
There will be quotes? I thought his used the `$OUTPUT_TASK_BUNDLE_LIST` that was placed by `save_ref`. I tried locally, got the `task-bundle-list` and there are no quotes within it, single reference per line, from what I can see. What am I missing?

Shouldn't this compute the `new_image`, something similar to what `strip_image_tag` is doing in `build-acceptable-bundles.sh` is doing?

Reply:
There will not be quotes. I originally had written the images to the file with quotes, but they were not needed so I removed them and didn't clean this. I'll push an update. 
It could compute `new_image` the same as `strip_image_tag`, but I thought it was better to have `build-acceptable-bundles` do it and pass that information to `push-and-tag`.